sdk/keyvault/azure-security-keyvault-keys/CHANGELOG.md

```diff
@@ -15,6 +15,7 @@
 ### Breaking Changes
 - Removed `EXPORT` from the `KeyOperation` enum.
 - Re-ordered parameters in the `EncryptResult` constructor to show `authenticationTag` before `additionalAuthenticatedData` to align with classes like `DecryptParameters`.
+- Removed service method overloads that take a `pollingDuration`, since `PollerFlux` and `SyncPoller` objects allow for setting this value directly on them.
```

Comment:
```suggestion
- Removed service method overloads that take a `pollingInterval`, since `PollerFlux` and `SyncPoller` objects allow for setting this value directly on them.
```